docs/usecase.rst

```diff
@@ -37,14 +72,37 @@ compute/data/communications abstractions,
 quality of service guarantees,
 and performance optimizations.
 
+.. seealso:: :doc:`executor`
+
 Generic sequential task execution
 ---------------------------------
 
 Units of work (nodes) describe the inputs and expected outputs,
 and identify a module able to perform the operation.
 
+| 1. Receive a node of work to execute.
```

Comment:
Not sure what 'node of work' means. Node as in a DAG?

Reply:
Node as in a DAG. I'm not completely happy with the term and welcome suggestions. I interchangeably use "node," "element," and "operation instance".

Reply:
I usually use 'unit of work' because I try to be agnostic towards a specific representation of the end user workflow. DAG Is the most common representation though so if we are assuming to use it explicitly, then I will understand what 'node of work' means in this context.

Reply:
I'm still cautious about 'unit of work' because it seems that many nodes may be decomposable into smaller units of work. "Unit of work" seems ambiguous with regard to scheduling. I do not yet see a reason not to assume a DAG, but I don't know if we want to insist that users think in such terms...